### ISSUE TITLE
Broken link to "Accuracy" in footer

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/FreeBMD2#449]
The link at the bottom of the footer (on t3 at least) points to "/cms/help#Accuracy" [sic]. There is no such `id` on the 'Help' page. We use lower-case or camelCase [sic] for in-page navigation.

The link now takes a user to the top of the Help page, so it is wounded rather than dead. However, it would be so much better to link to an actual id.

If we change the appropriate id on the Help page to 'Accuracy', sooner or later someone will wonder at the casing and change it to all lower-case.

I think that the best solution would be to change the id in the footer to "accuracy". 

Neither CEN nor REG has such a link.

**Comment by DeniseColbert** [comment]
@Vino-S to change the link ID in footer to accuracy (lower case)

**Comment by richpomfret** [comment]
Agreed to remove the link. Another story to be created to update the footer text.

**Comment by richpomfret** [comment]
Ready for testing on test 3.

**Comment by richpomfret** [comment]
@AlOneill could you take a look at/test this fix?

**Comment by AlOneill** [comment]
Link has been removed, so closing.

**Comment by DeniseColbert** [comment]
Has reappeared, reopening as link needs removing again (sorry @Vino-S !)

**Comment by DeniseColbert** [comment]
@DeniseColbert to test in test3

**Comment by KristinaGadzhieva** [comment]
For MVP change links in the footer:
- [x]  accuracy - link to the help page section;
- [x]  completeness - link to the records page 'Database coverage and statistics' section.

**Comment by Vino-S** [comment]
Ready for testing on test3

**Comment by Vino-S** [comment]
Deployed

**Comment by DeniseColbert** [comment]
Closing

**Comment by KristinaGadzhieva** [comment]
to change a link to the Help page

**Comment by KristinaGadzhieva** [comment]
@Vino-S please move out the explanation about Frequently Asked Questions

**Comment by KristinaGadzhieva** [comment]
- link needs to be updated;
- text needs to be updated, please see the screenshot

<img width="1126" height="860" alt="Image" src="https://github.com/user-attachments/assets/38e40614-8fbf-4ace-aa44-ba0a50e2d971" />

**Comment by KristinaGadzhieva** [comment]
please correct on test 3

**Comment by Vino-S** [comment]
Deployed along with the new Help

**Comment by KristinaGadzhieva** [comment]
We've checked in the meeting, same problem. @Vino-S could you please have a look at my last comment?

**Comment by richardofsussex** [comment]
I've addressed this issue.

**Comment by Vino-S** [comment]
Also I have updated the Accuracy link in beta

**Comment by Vino-S** [comment]
Deployed
